### ISSUE TITLE
unable to sync with prime france

### ISSUE THREAD

**Issue by pippo73** [trakt-tools/universal-trakt-scrobbler#409]
### Expected Behavior

sync with amazon prime

### Current Behavior

as I try to connect to amazon prime (france in my case) the app crashes

### Steps to Reproduce

install the plugin on chrome Version 0.4.1
select prime from the list

### Additional Information

- **OS + Version:** windows 11 

- **Browser + Version:** chrome Version 130.0.6723.92 (Build ufficiale) (a 64 bit)
- **Country/Region:** France

**Comment by chrisofspades** [comment]
I'm having a similar issue with Prime in the US. When I go to my Prime history the UI doesn't fully load. Inspecting the network tab shows the following error on requests to `/GetPlaybackResources` endpoint:

```
{
    "errorCode": "PRS.Dependency",
    "message": "Cannot complete request.",
    "type": "PRSDependencyException"
}
```

**Comment by Wikiwix** [comment]
I am unable to sync with Amazon Prime in Germany, based on the error logs (see below) I suspect that the page's html structure changed.
![image](https://github.com/user-attachments/assets/22a8569e-774f-4a61-ad0f-040b00ee1d0d)

**Comment by Harryb17** [comment]
> I am unable to sync with Amazon Prime in Germany, based on the error logs (see below) I suspect that the page's html structure changed. ![image](https://private-user-images.githubusercontent.com/5161388/385404420-22a8569e-774f-4a61-ad0f-040b00ee1d0d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI2NjY0NjQsIm5iZiI6MTczMjY2NjE2NCwicGF0aCI6Ii81MTYxMzg4LzM4NTQwNDQyMC0yMmE4NTY5ZS03NzRmLTRhNjEtYWQwZi0wNDBiMDBlZTFkMGQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjdUMDAwOTI0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmU0MzNiMzYwZjZjZTAxY2JlYjNjNGU2MmM1NWI1YzhiYmIyNTUxOTI4NzAxOTg0ODU0ZjY0ZTM0MzMzNmNjNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.DBcHUA0rDQWHhQfTVcyFS8bhVqVRPdjf80gTmHgLVj0)

I'm getting the exact same issue. Tried all debugging recommendations. 
Amazon Prime UK

**Comment by EremesNG** [comment]
> I'm having a similar issue with Prime in the US. When I go to my Prime history the UI doesn't fully load. Inspecting the network tab shows the following error on requests to `/GetPlaybackResources` endpoint:
> 
> ```
> {
>     "errorCode": "PRS.Dependency",
>     "message": "Cannot complete request.",
>     "type": "PRSDependencyException"
> }
> ```

It seems that the `/GetPlaybackResources` endpoint requires a value for `deviceID=`.  
I’m not sure if a specific value is needed, but in my tests, any number worked. Therefore, I made the following change, and it worked:

https://github.com/trakt-tools/universal-trakt-scrobbler/commit/ef1abb5e6ee6f20fcf9a8eb478e5166561b09088

![Captura de pantalla_8-12-2024_122344_mbhadeogepkjdjeikcckdkjdjhhkhlid](https://github.com/user-attachments/assets/cbb80b4a-f9bb-4701-b6bc-7a72c70357da)

**Comment by Wikiwix** [comment]
@MrMamen What is the plan around releasing the fix from #411?

**Comment by keldian** [comment]
Could someone please share their compiled build with the fix? Not sure why we have to wait so long for an official release.

**Comment by Wikiwix** [comment]
> Could someone please share their compiled build with the fix? Not sure why we have to wait so long for an official release.

@keldian I don't want to use my account's Trakt API access for a shared built. If you provide me a Client ID and Secret (see [here](https://github.com/trakt-tools/universal-trakt-scrobbler?tab=readme-ov-file#development)) I can share a built.

Be aware that you will loose all your settings though :(

**Comment by keldian** [comment]
I hadn't thought of API credentials. Never mind, I'll make my own build when I get around to it. Thanks though.

**Comment by Wikiwix** [comment]
@keldian Now there is an official beta release (with the fix) available: https://github.com/trakt-tools/universal-trakt-scrobbler/releases/tag/beta

**Comment by pippo73** [comment]
> [@keldian](https://github.com/keldian) Now there is an official beta release (with the fix) available: https://github.com/trakt-tools/universal-trakt-scrobbler/releases/tag/beta

I've tried this beta. It worked only once. The second time it crashed and since this moment it does not work anymore :-(

**Comment by keldian** [comment]
Works consistently on my end. Edge 133.0.3065.82, Windows 11 24H2

**Comment by mwilkinson22** [comment]
I've just installed 0.5.0 thinking this fix would be included but unfortunately I'm still getting the same error (Prime UK). Is this expected, or is the fix not working for me?

**Comment by keldian** [comment]
> I've just installed 0.5.0 thinking this fix would be included but unfortunately I'm still getting the same error (Prime UK). Is this expected, or is the fix not working for me?

Still works for me (Prime FR) with 0.5.0

Keep in mind you need to be signed in to Prime on the same browser, with your profile selected. Sessions expire so you have to repeat this from time to time.

**Comment by mwilkinson22** [comment]
Thanks @keldian - but looks like I am logged in. I'm getting this error in the console, which is different to the one reported by Wikiwix (not sure if it's a similar issue). I can open a new issue if that's easier

I'd add that the error suggests a failing API call - but there's nothing in the Network tab, so it looks like the request isn't actually being made

<img width="1406" height="800" alt="Image" src="https://github.com/user-attachments/assets/bc296e70-759c-466d-99cf-c8f4a6d18ae0" />

**Comment by keldian** [comment]
@mwilkinson22 I would take that to a new issue since this one specifies "Prime France" and the problem could be UK specific. I would also suggest copy & pasting error messages rather than posting screenshots.

**Comment by MrMamen** [comment]
Fixed in #449
